ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/AbstractBlockProcessor.java

```diff
@@ -164,7 +164,7 @@ protected AbstractBlockProcessor(
               blockHeader,
               transaction,
               miningBeneficiary,
-              OperationTracer.NO_TRACING,
+              OpenTelemetryTracing.INSTANCE,
```

Comment:
I have performance concerns about hard coding anything but a no-op tracer in this loop.  Can it be made CLI configurable and default to `NO_TRACING`?

Reply:
Yes it can. Let's do it.